lib/src/block_syntaxes/alert_block_syntax.dart

```diff
@@ -1,110 +0,0 @@
-// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
-// for details. All rights reserved. Use of this source code is governed by a
-// BSD-style license that can be found in the LICENSE file.
-
-import '../ast.dart';
-import '../block_parser.dart';
-import '../line.dart';
-import '../patterns.dart';
-import 'block_syntax.dart';
-import 'code_block_syntax.dart';
-import 'paragraph_syntax.dart';
-
-/// Parses GitHub Alerts blocks.
-///
-/// See also: https://docs.github.com/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts
-class AlertBlockSyntax extends BlockSyntax {
-  const AlertBlockSyntax();
-
-  @override
-  RegExp get pattern => alertPattern;
-
-  @override
-  bool canParse(BlockParser parser) {
-    return pattern.hasMatch(parser.current.content) &&
-        parser.lines.any((line) => _contentLineRegExp.hasMatch(line.content));
-  }
-
-  /// Whether this alert ends with a lazy continuation line.
-  ///
-  /// The definition of lazy continuation lines:
-  /// https://spec.commonmark.org/0.30/#lazy-continuation-line
-  static bool _lazyContinuation = false;
-  static final _contentLineRegExp = RegExp(r'>?\s?(.*)*');
-
-  @override
-  List<Line> parseChildLines(BlockParser parser) {
-    // Grab all of the lines that form the alert, stripping off the ">".
-    final childLines = <Line>[];
-    _lazyContinuation = false;
-
-    while (!parser.isDone) {
-      final strippedContent =
-          parser.current.content.replaceFirst(RegExp(r'^\s*>?\s*'), '');
-      final match = strippedContent.isEmpty
-          ? null
-          : _contentLineRegExp.firstMatch(strippedContent);
```

Comment:
This makes it unable to distinguish between an empty line and a line containing only `>`, but these are very different at result in different behavior.